plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/_impl.py

```diff
@@ -201,8 +200,7 @@ def sweep(self, arguments: List[str]) -> None:
             returns = self.launcher.launch(overrides, initial_job_idx=self.job_idx)
             self.job_idx += len(returns)
             for trial, ret in zip(trials, returns):
-                # TODO (toshihikoyanase): Remove type-ignore when optuna==2.4.0 is released.
-                study._tell(trial, optuna.trial.TrialState.COMPLETE, ret.return_value)  # type: ignore
+                study._tell(trial, optuna.trial.TrialState.COMPLETE, ret.return_value)
```

Comment:
`study._tell` expects a list of values because Optuna v2.4.0 changes the study class to supports multi-objective optimization. c.f., https://github.com/optuna/optuna/blob/b4595a6c17cc115ce6299b13ab5db8ce8d23522a/optuna/study.py#L668-L670
```suggestion
                study._tell(trial, optuna.trial.TrialState.COMPLETE, [ret.return_value])
```